lib/Conversion/SCFToCalyx/SCFToCalyx.cpp

```diff
@@ -396,9 +396,19 @@ class BuildOpGroups : public calyx::FuncOpPartialLoweringPattern {
   template <typename TGroupOp, typename TCalyxLibOp, typename TSrcOp>
   LogicalResult buildLibraryOp(PatternRewriter &rewriter, TSrcOp op,
                                TypeRange srcTypes, TypeRange dstTypes) const {
+    auto castToInteger = [](Type type) -> Type {
```

Comment:
Nit: maybe `toBitVector` or `bitCast` or something similar to make it clear we're performing a bit cast and not changing the value.